cluster/images/provider-terraform-controller/.gitconfig

```diff
@@ -1,2 +1,2 @@
 [credential]
-	helper = store --file=$GIT_CRED_DIR/.git-credentials
+	helper = store --file=$GIT_CRED_DIR/.git-credentials
```

Comment:
https://thoughtbot.com/blog/no-newline-at-end-of-file#history-lesson please keep the new line